src/components/RecipeBox.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+export class RecipeBox extends Component {
+	render() {
+		return (
+			<div className="recipeBox">
+				<img src={this.props.image}></img>
```

Comment:
Today we discussed the need for https://facebook.github.io/react/docs/reusable-components.html#prop-validation